.circleci/config.yml

```diff
@@ -52,6 +52,10 @@ jobs:
           name: Run tests
           command: cargo test --release
           no_output_timeout: 30m
+      - run:
```

Comment:
I recall there was a tool/script which would `cargo <command>` with all possible features combinations, but I cannot find it

Reply:
If you do, I'd love to use it